nucypher/cli/commands/stake.py

```diff
@@ -524,6 +527,80 @@ def create(general_config, transacting_staker_options, config_file, force, value
     paint_staking_confirmation(emitter=emitter, staker=STAKEHOLDER, receipt=receipt)
 
 
+@stake.command()
+@group_transacting_staker_options
+@option_config_file
+@option_force
+@option_value
+@option_index
+@group_general_config
+def increase(general_config, transacting_staker_options, config_file, force, value, index):
+    """Increase an existing stake."""
+
+    # Setup
+    emitter = setup_emitter(general_config)
+    STAKEHOLDER = transacting_staker_options.create_character(emitter, config_file)
+    blockchain = transacting_staker_options.get_blockchain()
+    economics = STAKEHOLDER.economics
+
+    client_account, staking_address = select_client_account_for_staking(
+        emitter=emitter,
+        stakeholder=STAKEHOLDER,
+        staking_address=transacting_staker_options.staker_options.staking_address,
+        individual_allocation=STAKEHOLDER.individual_allocation,
+        force=force)
+
+    # Handle stake update and selection
+    if index is not None:  # 0 is valid.
+        current_stake = STAKEHOLDER.stakes[index]
+    else:
+        current_stake = select_stake(staker=STAKEHOLDER, emitter=emitter)
+
+    #
+    # Stage Stake
+    #
+
+    if not value:
+        token_balance = STAKEHOLDER.token_balance
```

Comment:
If token_balance is 0 (which was happening to me before because Felix was stuck) the command should exit here with some error message.

Reply:
✔️